discord/interactions.py

```diff
@@ -137,6 +136,7 @@ class Interaction:
         "custom_id",
         "_message_data",
         "_permissions",
+        "_app_permissions",
         "_state",
         "_session",
         "_original_message",
```

Comment:
```suggestion
        "_original_message",
        "_cs_app_permissions",
```
Neccesary for the below change to function